File: dofun/safe/deviceinfo/info/BatteryInfo.java
package com.dofun.safe.deviceinfo.info;

import android.content.Context;
import androidx.core.util.Pair;
import com.dofun.safe.deviceinfo.utils.BatteryUtils;
import com.dofun.safe.deviceinfo.utils.CommandUtils;
import java.util.ArrayList;
import java.util.List;
import org.json.JSONObject;

/* loaded from: classes8.dex */
public class BatteryInfo {
    public static List<Pair<String, String>> getBatteryInfo(Context context) {
        ArrayList<Pair<String, String>> list = new ArrayList<>();
        BatteryUtils.getBatteryInfo(context, list);
        return list;
    }

    public static JSONObject getBatteryInfoJson(Context context) {
        return CommandUtils.convertToJson(getBatteryInfo(context));
    }
}

File: app/src/main/java/com/dofun/safe/deviceinfo/info/BatteryInfo.kt
package com.dofun.safe.deviceinfo.info

import android.content.Context
import androidx.core.util.Pair
import com.dofun.safe.deviceinfo.utils.BatteryUtils
import com.dofun.safe.deviceinfo.utils.CommandUtils
import org.json.JSONObject

object BatteryInfo {
    @JvmStatic
    fun getBatteryInfo(context: Context): List<Pair<String, String>> {
        val list = ArrayList<Pair<String, String>>()
        BatteryUtils.getBatteryInfo(context, list)
        return list
    }

    @JvmStatic
    fun getBatteryInfoJson(context: Context): JSONObject {
        return CommandUtils.convertToJson(getBatteryInfo(context))
    }
}
